modular_citadel/code/modules/donatorreskins/reskinkits.dm

```diff
@@ -8,6 +8,24 @@
 	icon_state = "box" //icon_state of the KIT, not the reskinned item
 	var/product = /obj/item/clothing/head/helmet/space/void/security/jenna //what it makes
 	var/list/fromitem = /obj/item/clothing/head/helmet/space/void/security //what it needs
+	afterattack(obj/O, mob/user as mob) //after being used
```

Comment:
Relative pathing is not allowed, and don't use "as variable" unless it's unclear what variable type it is (it's pretty clear mob/user should be any /mob or subtype of /mob)